docs/api/sm-api.md

```diff
@@ -279,6 +284,19 @@ which returns
 }
 ```
 
+To sort by descending `sent_date` followed by ascending `sender_name`:
+```
+sort[]=-sent_date&sort[]=sender_name
```

Comment:
ahh, nm, you've got that documented here, but not clear why you have the kebab case one still.